components/definitions/chromel.d.ts

```diff
@@ -69,7 +69,7 @@ declare namespace chrome.braveRewards {
   const getPublisherInfo: (publisherKey: string, callback: (result: RewardsExtension.Result, properties: RewardsExtension.PublisherInfo) => void) => {}
   const getPublisherPanelInfo: (publisherKey: string, callback: (result: RewardsExtension.Result, properties: RewardsExtension.PublisherInfo) => void) => {}
   const savePublisherInfo: (windowId: number, mediaType: string, url: string, publisherKey: string, publisherName: string, favIconUrl: string, callback: (result: RewardsExtension.Result) => void) => {}
-  const tipSite: (tabId: number, publisherKey: string, monthly: boolean) => {}
+  const tipSite: (tabId: number, publisherKey: string, entryPoint: 'one-time' | 'set-monthly' | 'clear-monthly') => {}
```

Comment:
Should this be its own type? I saw we reused it elsewhere, it might help keep them in sync.